### PR TITLE
2.1.1 (Foxy)

### PR DESCRIPTION
**Pull request by tylerjw**
This is a problem because 2.1.1 was released for Rolling and Galactic but never for foxy.  I'm not sure what to do about that as we should probably release these changes into foxy.

### CODE REVIEW

**Comment by rhaschke** [comment]
Why do you need a separate foxy branch at all? As far as I can see (but I checked a few files only), the code differences are limited to the copyright notice. What about simply merging and continuing with a singly ros2 branch?

**Comment by tylerjw** [comment]
Based on the email I got from the buildfarm it is yet another one of the .h/.hpp incldes.  I'm going to make a PR against the ros2 branch that should make this build for foxy.

File: CHANGELOG.rst [hunk 1]
```diff
@@ -2,6 +2,18 @@
 Changelog for package geometric_shapes
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+2.1.1 (2021-12-22)
+------------------
+* Fix cmake such that Boost::filesystem is exported properly (`#206 <https://github.com/ros-planning/geometric_shapes/issues/206>`_) (`#211 <https://github.com/ros-planning/geometric_shapes/issues/211>`_)
+  Co-authored-by: Jafar Abdi <cafer.abdi@gmail.com>
```
Comment (henningkayser) [inline CHANGELOG.rst:8]: Co-authors can be dropped here and listed under "Contributors", imo. Also, do we want to keep/drop CI changes from the changelog in the future?

**Review by henningkayser — APPROVED** [review]
Looks good, fixing my comment is optional. Feel free to merge and push the tag.

**Comment by henningkayser** [comment]
> Based on the email I got from the buildfarm it is yet another one of the .h/.hpp incldes. I'm going to make a PR against the ros2 branch that should make this build for foxy.

Maybe we were a little to fast with creating these branches. In the future they might still be useful, and syncing the changes here should be trivial.

**Comment by tylerjw** [comment]
@henningkayser look at the other PR for adding support for Foxy to the ros2 branch.  I think we should close this as it has a SemVar issue and we should be able to release from R/G/F all from the ros2 branch after that other PR.  Then after the release is synced we can delete this branch (buildfarm doesn't keep old versions anyway).

**Comment by henningkayser** [comment]
> @henningkayser look at the other PR for adding support for Foxy to the ros2 branch. I think we should close this as it has a SemVar issue and we should be able to release from R/G/F all from the ros2 branch after that other PR. Then after the release is synced we can delete this branch (buildfarm doesn't keep old versions anyway).

Yeah, just wanted to say that I noticed the other one right now ;)

**Pull request closed without merging**